src/test_project/test_fixtures/src/lib.rs

```diff
@@ -1,3 +1,4 @@
+//! This is a test project for racer.
```

Comment:
We've had issues before with multiline comments that include paragraph-separator lines. Given that crate docs are disproportionately likely to be long and have code samples, can you flesh out this comment to exercise:

* Paragraph break
* Code block, including a doc comment like this one inside that example block?